db/seeds.rb

```diff
@@ -4,14 +4,14 @@
 include FactoryGirl::Syntax::Methods
 require 'timecop'
 
-%w[Cane Crutches Wheelchair].each do |device|
-  create :mobility_device, device: device
-end
-
 2.times { create :user, :admin }
 
 5.times { create :user }
 
+%w[Cane Crutches Wheelchair].each do |device|
```

Comment:
Why are there only three? I thought there were four or five?

Reply:
It's seeds, so not super important — just to simulate real data. But I can add a few more if you like.

Reply:
Nah, thanks for clarifying.